src/App/Controls/Settings/BaseSettingControlView.cs

```diff
@@ -1,14 +1,24 @@
-﻿using Xamarin.Forms;
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Bit.App.Utilities;
+using Xamarin.Forms;
+using static PInvoke.BCrypt;
 
 namespace Bit.App.Controls
 {
     public class BaseSettingItemView : ContentView
     {
         public static readonly BindableProperty TitleProperty = BindableProperty.Create(
-            nameof(Title), typeof(string), typeof(SwitchItemView), null, BindingMode.OneWay);
+            nameof(Title), typeof(string), typeof(SwitchItemView), null);
 
         public static readonly BindableProperty SubtitleProperty = BindableProperty.Create(
-            nameof(Subtitle), typeof(string), typeof(SwitchItemView), null, BindingMode.OneWay);
+            nameof(Subtitle), typeof(string), typeof(SwitchItemView), null);
+
+        public static readonly BindableProperty ItemEnabledProperty = BindableProperty.Create(
+            nameof(ItemEnabled), typeof(bool), typeof(SwitchItemView), true);
+
+        public static readonly BindableProperty ItemTextColorProperty = BindableProperty.Create(
+            nameof(ItemTextColor), typeof(Color), typeof(SwitchItemView));
```

Comment:
🎨 I think this is not needed. You can use the `IsEnabled` property directly, and in the `ControlTemplate` use a converter that depends on the is enabled bool to change the color.
Just FYI, when you want to do something on some bindable property changing, you can directly use the parameter of `propertyChanged` of the `BindableProperty.Create` instead of the general one of the control so you don't have to check whether the property is the one you want.